recipes/panzi_portable_endian/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+import os
+from conans import ConanFile, tools
+
+
+class PanziPortableEndian(ConanFile):
+    name = "panzi_portable_endian"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gist.github.com/panzi/6856583"
+    description = "This provides the endian conversion functions form endian.h on Windows, Linux, *BSD, and Mac OS X"
+    topics = ("conan", "endian")
+    license = "Public Domain"
```

Comment:
```suggestion
    license = "Unlicense"
```

Public Domain is not a license name, but a group of licenses. It's not clear which license is using, but let's follow Unlicense

Reply:
fixed